Userland/Applications/TodoList/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+serenity_component(
+    TodoList
+    REQUIRED
```

Comment:
This app is probably not _required_ for a minimal system.

Reply:
Ah ah, I copy-pasted the cmake file structure from some other app without thinking about it 🙂